### PR TITLE
fix docs error links

### PR DESCRIPTION
**Pull request by xiaoping378**
* [x] Updated all relevant documentation in docs
* [x] Updated all code comments where relevant
* [x] Wrote tests
* [x] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by melekes** [comment]
@zramsay can you take a look at this please?

File: docs/DOCS_README.md [hunk 1]
```diff
@@ -4,9 +4,15 @@ The documentation found in this directory is hosted at:
...
+and built using [VuePress](https://vuepress.vuejs.org/) like below:
 
-- https://github.com/tendermint/tendermint.com
+```bash
+npm install -g vuepress # global install vuepress tool, only once
+npm install
+
+mkdir -p .vuepress && cp config.js .vuepress/
+vuepress build
+```
```
Comment (zramsay) [inline docs/DOCS_README.md]: nice 👍

File: docs/DOCS_README.md [hunk 2]
```diff
@@ -4,9 +4,15 @@ The documentation found in this directory is hosted at:
...
 
 Under the hood, Jenkins listens for changes (on develop or master) in ./docs then rebuilds
 either the staging or production site depending on which branch the changes were made.
```
Comment (zramsay) [inline docs/DOCS_README.md]: Can you add a sentence that says:

To view the latest documentation on the develop branch, see the staging site:

- https://tendermint-staging.interblock.io/docs/

and the documentation on master branch is found here:

- https://tendermint.com/docs/

File: docs/tendermint-core/light-client-protocol.md [hunk 3]
```diff
@@ -11,10 +11,10 @@ package](https://godoc.org/github.com/tendermint/tendermint/lite).
 
 The objective of the light client protocol is to get a
 [commit](./validators.md#committing-a-block) for a recent [block
-hash](../spec/consensus/consensus.md.md#block-hash) where the commit includes a
+hash](./block-structure.md#block-hash) where the commit includes a
```
Comment (zramsay) [inline docs/tendermint-core/light-client-protocol.md]: remove the `#block-hash` part (see recently updated block-structure.md)

Comment (zramsay) [reply docs/tendermint-core/light-client-protocol.md]: actually, remove the whole link

File: docs/tendermint-core/light-client-protocol.md [hunk 4]
```diff
@@ -10,11 +10,11 @@ package](https://godoc.org/github.com/tendermint/tendermint/lite).
 ## Overview
 
 The objective of the light client protocol is to get a
-[commit](./validators.md#committing-a-block) for a recent [block
-hash](../spec/consensus/consensus.md.md#block-hash) where the commit includes a
+[commit](./validators.md#committing-a-block) for a recent block
```
Comment (zramsay) [inline docs/tendermint-core/light-client-protocol.md]: `commit-a-block`, like above

**Comment by melekes** [comment]
Thanks for contributing @xiaoping378 !

**Pull request merged**